server/container_create_linux.go

```diff
@@ -757,6 +757,26 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrfactory.Cont
 		unshare.IsRootless(),
 		ctr.DisableFips(),
 	)
+	if ctr.DisableFips() {
+		// Create a temporary file to store the value to be written to /proc/sys/crypto/fips_enabled
+		tmpFile, err := os.CreateTemp(containerInfo.RunDir, "tmpfips-")
```

Comment:
No need to use `CreateTemp` here, you can just write a file with a unique name like `sysctl-fips`.

Also, you forgot to close the file.

Replace the whole thing with `os.WriteFile` so you don't have to create/write/close.